go/cmd/vtcombo/main.go

```diff
@@ -76,6 +79,30 @@ func main() {
 	// set discoverygateway flag to default value
 	flag.Set("cells_to_watch", strings.Join(tpb.Cells, ","))
 
+	flag.Set("cell", tpb.Cells[0])
```

Comment:
Ah I missed your comment. It's probably because the GetKnownCells call is not handled properly. I'll fix that eventually, please add a FIXME to remove this.